src/routes/thumbnail/+page.svelte

```diff
@@ -0,0 +1,125 @@
+<script lang="ts">
+	import GitHubAvatar from '$lib/components/GitHubAvatar.svelte';
+	import html2canvas from 'html2canvas';
+
+	let title = 'Your talk title, it will be awesome!';
+	let githubAuthor = 'jycouet';
+	let name = '';
+
+	async function dl() {
+		const el = document.querySelector('#tumb');
+		if (el) {
+			let proxy = '';
+
+			const existingGithubAuthorsForPicture = ['dreitzner', 'jjnp', 'jycouet'];
+			if (existingGithubAuthorsForPicture.includes(githubAuthor)) {
+				// We have to have it locally if not we can't download :( the image with the picture.
+				// HELP: Maybe someone know how to get it in the final image with the github picture directly?
```

Comment:
https://github.com/niklasvh/html2canvas/issues/722#issuecomment-166824752

maybe this?